crates/aoe2rec/src/header/mod.rs

```diff
@@ -443,15 +449,23 @@ pub struct PlayerInit {
     pub culture: u8,
     pub civilization: u8,
     pub game_status: u8,
-    #[br(pad_after = 1)]
+    //#[br(dbg, pad_after = 1)]
     pub resigned: Bool,
-    #[br(pad_after = 1)]
+    unk5: u8,
+
     pub player_color: u8,
+    unk6: u8,
+
     #[serde(skip_serializing)]
-    #[br(magic = b"\x00\x0B")]
+    //#[br(magic = b"\x00\x0B")]
```

Comment:
These need to be there because changes in the rec structure are often caught by such magic numbers